### PR TITLE
Remove ecs_task_family_name from repo

### PR DESCRIPTION
**Pull request by FyzHsn**
- [ ] I have added the tests to cover my changes.
- [ ] I have updated the documentation and CHANGELOG accordingly.
- [ ] I have read the CONTRIBUTING document.

### CODE REVIEW

File: covalent_ecs_plugin/ecs.py [hunk 1]
```diff
@@ -187,10 +183,10 @@ async def submit_task(self, task_metadata: Dict, identity: Dict) -> Any:
 
         # Register the task definition
         self._debug_log("Registering ECS task definition...")
-        self.ecs_task_family_name = f"{dispatch_id}-{node_id}"
+        self._ecs_task_family_name = f"{dispatch_id}-{node_id}"
```
Comment (AlejandroEsquivel) [inline covalent_ecs_plugin/ecs.py:186]: Given this is not used anywhere else @FyzHsn let's just make this a local variable
```suggestion
ecs_task_family_name = f"{dispatch_id}-{node_id}"
```

File: covalent_ecs_plugin/ecs.py [hunk 2]
```diff
@@ -129,7 +125,7 @@ def __init__(
         )
         self.vcpu = vcpu or get_config("executors.ecs.vcpu")
         self.memory = memory or get_config("executors.ecs.memory")
-        self._cwd = tempfile.mkdtemp()
+        self._ecs_task_family_name = ""
```
Comment (AlejandroEsquivel) [inline covalent_ecs_plugin/ecs.py:128]: Thanks for including this much-needed change re: `_cwd` @FyzHsn , while we are at it, can we move `L133` which creates the cache_dir to outside the constructor of the executor for self-hosted support? Either the top-most line in the `run` method works or if we create a getter method which also creates it if it doesn't exist. TIA!

**Review by AlejandroEsquivel — CHANGES_REQUESTED** [review]
LGTM , thanks for including the _

File: covalent_ecs_plugin/ecs.py (hunk 1 above)
Comment (FyzHsn) [reply covalent_ecs_plugin/ecs.py:186]: Ah I thought about that @AlejandroEsquivel but that value is used later on in `get_status`.

**Review by AlejandroEsquivel — APPROVED** [review]
thanks @FyzHsn looks good 👍🏼

**Pull request merged**